packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -11,7 +11,7 @@ import { AMM_INSTANCE } from './params.js';
 const makeLiquidityKeyword = keyword => `${keyword}Liquidity`;
 
 /**
- * Collateral Reserve holds onto Collateral for the RUN protocol, and can
+ * Asset Reserve holds onto assets for the RUN protocol, and can
```

Comment:
renaming the collateralReserve seems unrelated to the point of this PR.

Reply:
Good call. I'll move 34a45e3 to 5012-move-penalty-funds-to-reserve